numba/core/errors.py

```diff
@@ -544,7 +544,8 @@ class NotDefinedError(IRError):
 
     def __init__(self, name, loc=None):
         self.name = name
-        msg = "Variable '%s' is not defined." % name
+        msg = ("The compiler failed to analyze the bytecode. "
+               "Variable '%s' is not defined." % name)
```

Comment:
Is this the result of a merge that's gone awry? Fairly sure I saw a patch with this in recently.

Reply:
It is indeed... I wrote that patch you mentioned and I ended up merging it on my local master too and this happened